plugins/generator-1.18.2/forge-1.18.2/procedures/entity_islooking_at_entity.java.ftl

```diff
@@ -0,0 +1,15 @@
+(new Object() {
+	public boolean isLooking(Entity _source, Entity _target, double area, boolean ignoreBlocks) {
+		Vec3 vec31 = new Vec3(_target.getX() - _source.getX(), (_target.getEyeY() - _target.getEyeHeight() * 0.5) - _source.getEyeY(),

```

Comment:
What is the purpose of `_target.getEyeHeight()`?

Also, what is this code based on? What was to inspiration/source of this code?

Reply:
Eyeheight is to then divide it by 2 and get the middle of the entity, although now that you mention it, using height would be better; It's based on a modified  endermen code

Reply:
Depends if looking at is considered middle or eyes of the entity, I guess it is fine, but yep, use full entity height